app/controllers/transactions_controller.rb

```diff
@@ -0,0 +1,36 @@
+class TransactionsController < ApplicationController
+
+  customer_tab :all
+  before_action :authenticate_user!
+  before_action :check_acting_as
+
+  include TransactionSearch
+
+  def initialize
+    @active_tab = "accounts"
+    super
+  end
+
+  def index_with_search
+    @recently_reviewed = administered_order_details.recently_reviewed.paginate(page: params[:page])
+    @order_details = administered_order_details.all_in_review
+    @extra_date_column = :reviewed_at
+    @order_detail_link = {
+      text: text("shared.dispute"),
+      display?: proc { |order_detail| order_detail.can_dispute? },
+      proc: proc { |order_detail| order_order_detail_path(order_detail.order, order_detail) },
```

Comment:
I took some time on this, and it's leading to a bigger refactor than I expected, mostly due to the views involved. I think it's worth doing, but I'd like to put it in a separate PR.